### ISSUE TITLE
Assets folder does not appear as source folder eclipse

### ISSUE THREAD

**Issue by thechubbypanda** [libgdx/libgdx#5238]
#### Issue details
As the Gradle files stand, the assets folder is not automatically set as a source folder for the project. This can be fixed by adding it to the build path in eclipse however a gradle refresh puts the problem back.

#### Reproduction steps/code
jre version 9.0.4

Open gdx-setup.jar
untick android, ios and html
tick freetype
advanced ---> tick eclipse
generate

---- In Eclipse ----
Import ---> gradle ---> Existing gradle project
Finish
Run desktop as java application

#### Version of LibGDX and/or relevant dependencies
1.9.8

#### Stacktrace
```java
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load file: badlogic.jpg
	at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:149)
	at com.badlogic.gdx.graphics.TextureData$Factory.loadFromFile(TextureData.java:98)
	at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:100)
	at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:92)
	at com.badlogic.gdx.graphics.Texture.<init>(Texture.java:88)
	at net.thechubbypanda.larrysadventure.Game.create(Game.java:16)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:149)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: File not found: badlogic.jpg (Internal)
	at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:136)
	at com.badlogic.gdx.files.FileHandle.readBytes(FileHandle.java:222)
	at com.badlogic.gdx.graphics.Pixmap.<init>(Pixmap.java:146)
	... 7 more

```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [x] Windows
- [ ] Linux
- [ ] MacOS

**Comment by mgsx-dev** [comment]
The workaround i always use is to change the working directory of eclipse desktop launcher to the asset folder : in launcher configuration / arguments tab / working directory / Other / Workspace...

This also avoid to have to refresh a linked folder whenether some assets have changed ;-)

**Comment by Tom-Ski** [comment]
Best thing to do is to use the working directory. We did have asset folder linked previously, so Ill see what has changed so we can revert back to previous behavior.

**Comment by mgsx-dev** [comment]
I made several tests trying to solve the issue : 

* when generating project for eclipse and importing as eclipse project, it works : external asset folder is present and in classpath.
* when generating project for no IDE and importing as gradle project in eclipse : external folder is present but not in classpath. It's required to run afterEclipseImport gradle task manually and refresh project (F5). This is another workaround which have to be done manually after a gradle project refresh.

Unfortunately it can't be automated since gradle eclipse plugin (plugin installed into eclipse) doesn't provide a way to attach tasks to the refresh process (see here : https://github.com/eclipse/buildship/issues/265)

Finally i tried to modify setup tool this way : 

* set external asset folder as resource like this : `sourceSets.main.resources.srcDirs = [ assetsDir ]`
* totally removed afterEclipseImport task.
* I removed asset folder inclusion from desktop:dist task to avoid file duplication in jar.

Finally all work well for both scenario.

I'll PR my changes in order to see if it's fine, if i didn't missed something important...
